package.json

```diff
@@ -24,9 +25,11 @@
     "test:size": "bundlesize",
     "flow": "flow check",
     "flow:watch": "flow-watch",
+    "format": "yarn run prettier && eslint src --fix",
```

Comment:
It's a big advantage to run prettier as part of the eslint plugin for multiple reasons:

- Integration of the two
- Avoid duplicated/staggered errors
- Avoid confusing order of prettier & eslint (here eslint runs after prettier, so there's still a chance of conflicts if we change anything in the eslint config)
- Integration into editors that have eslint integration but not prettier integration
- And a big one: get all messages in the same formatted way

Reply:
Thanks for the feedback! To ensure I understand, we want to:

- Re-add `eslint-plugin-prettier`, exactly as was implemented by @existentialism 
- Set the pre-commit hook to run `eslint src --fix`. With `eslint-plugin-prettier`, this script will automatically run `prettier`
- Add `eslint-plugin-flowtype` to .eslintrc

Is this correct?

Reply:
> Add eslint-plugin-flowtype to .eslintrc

The plugins should be there, rather we'll want to add the eslint-config-prettier/flow config. There's more info on the available configs here: https://github.com/prettier/eslint-config-prettier

> If so - we still need to keep eslint-config-prettier, right?

That'd be great, yea :+1: